### DISCUSSION TITLE
Bug Report cargohuskcult mission no longer exists in my files

### DISCUSSION THREAD

**Discussion by NeuraxWolf** [FakeFishGames/Barotrauma#13709] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

church of the husk mission is not getable, when asking the people for it nothing happens except i get a message in control panel saying it does not exist
Edit1 4/7/24: something edited what the game looks for, the real mission name is really "cargohuskeggs" but something made my game look for "cargohuskcult"

### Reproduction steps

1. start solo campaign
2. find husk church quest giver and ask for their cargo mission

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.3.0.0/v1.3.0.1

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
if told how to get a report i will get one
```

**Comment by SomeRandomNoobKekeke** [comment]
Hmm...
i can "unlockmission cargohuskeggs" and "unlockmission escort1huskcult"
what mission do you mean? is it some moded mission?

**Reply by NeuraxWolf** [reply]
huh... weird unlockmission cargohuskeggs gives the mission but whenever i get it naturally i get this:
![image_2024-04-01_023417997](https://github.com/FakeFishGames/Barotrauma/assets/46556230/67e46394-fbf9-4439-9e4c-cf896dd96d06)

**Reply by SomeRandomNoobKekeke** [reply]
Could you give me that save?

**Reply by NeuraxWolf** [reply]
How may i do that? id love to!

**Reply by SomeRandomNoobKekeke** [reply]
What? give me the save? go to "C:\Users\user\AppData\Local\Daedalic Entertainment GmbH\Barotrauma"
grab your .save file, zip it and drop here

**Reply by NeuraxWolf** [reply]
alright i assume you meant zip the save file?
[woofers2.zip](https://github.com/FakeFishGames/Barotrauma/files/14896970/woofers2.zip)

**Reply by SomeRandomNoobKekeke** [reply]
Ye, i mean, github doesn't allow you to share unzipped files

I'd suspect Better Pay (More Marks), because it does change missions, try disabling it
Somehow i don't even get husk module in that save, there's just clowns, idk why

**Reply by NeuraxWolf** [reply]
you were right :( but i wish that wasnt the problematic mod

**Reply by SomeRandomNoobKekeke** [reply]
Barodevs will add mission reward multiplier to campaign settings in next update

**Comment by Regalis11** [comment]
I'm afraid I could not reproduce the issue, the mission seems to unlock with no errors:
![image](https://github.com/FakeFishGames/Barotrauma/assets/14351070/aa60e20c-3a3e-49bd-a749-454932cd8c96)

You mentioned you're using mods, could it be that some of them modify the missions in some way? The error message suggests that there's no mission with the tag "cargohuskcult", but if unlocking the mission by identifier still works, it's likely that some mod modifies the tags of that mission in some way.

**Reply by NeuraxWolf** [reply]
maybe? i don't see how any could modify the tag of the mission
